src/main/java/org/opentripplanner/routing/api/request/framework/DoubleAlgorithmFunction.java

```diff
@@ -0,0 +1,13 @@
+package org.opentripplanner.routing.api.request.framework;
+
+import java.io.Serializable;
+
+/**
+ * OTP support injecting custom functions into the algorithm. These functions
+ * must support this interface.
+ */
```

Comment:
```suggestion
/**
 * OTP support injecting custom functions into the algorithm. These functions
 * must support this interface.
 * <p>
 * The implementation must be Serializable and have a nice toString(), like: {@code f(x)= 2.3 + 1.33 * x} 
 * <p>
 * Use the {@link org.opentripplanner.routing.api.request.framework.RequestFunctions} to create new objects 
 * of this type.
 */
```

Reply:
I think we should rename `RequestFunctions` to `AlgorithmFunctionFactory`, is the name for `DoubleAlgorithmFunction` ok?